extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -375,6 +440,14 @@ class AmpLightbox extends AMP.BaseElement {
     this.triggerEvent_(LightboxEvents.CLOSE);
   }
 
+  /**
+   * @return {boolean}
+   * @private
+   */
+  isInAd_() {
```

Comment:
or, can we just check if the element is in an iframe? Is there a way to check if the owner document is the one AMP runtime is on?

Reply:
Done.

Reply:
Reverted.